### ISSUE TITLE
Assessment Feedback

### ISSUE THREAD

**Issue by mnhollandplum** [jaw772/mod_0_assessment#1]
Hey Josh! Great work on your assessment and your multiple choice quiz. I like that you practiced committing in super small chunks! Let me know if you have any questions.

**Comment by jaw772** [comment]
Thanks. I honestly thought I messed up and did it incorrectly on the first part where I committed after every change I made to the file. I thought this because the instructions didn't say to save and commit until the end. So I didn't commit in small chunks on the second half.

**Comment by mnhollandplum** [comment]
Yeah, technically it was not part of the assignment but it’s good practice regardless. In the future you won’t need to do commits that often.
